src/components/rangeslider/index.js

```diff
@@ -0,0 +1,292 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Plotly = require('../../plotly');
+
+var Lib = require('../../lib');
+var Plots = require('../../plots/plots');
+
+var helpers = require('./helpers');
+var rangePlot = require('./range_plot');
+var attributes = require('./attributes');
+
+var svgNS = 'http://www.w3.org/2000/svg';
+
+module.exports = {
+    draw: draw,
+    supplyLayoutDefaults: supplyLayoutDefaults
+};
+
+function draw(gd, minStart, maxStart) {
+    var fullLayout = gd._fullLayout,
+        options = fullLayout.xaxis.rangeslider;
+
+    // don't draw if being exported
+    if(!options.visible || gd._context.staticPlot) {
+        options.visible = false;
+        return;
```

Comment:
It happily exports it :)

We could maybe add the option to have it displayed on export or not as an attribute too. 

`excludeonexport: false` or something a bit less verbose.